### PR TITLE
NET-225 Replace static classes with enum if possible and fix naming

### PR DESCRIPTION
**Pull request by BohdanBuhrii**

### CODE REVIEW

File: TransIT.BLL/Helpers/Roles.cs [hunk 1]
```diff
@@ -1,13 +1,13 @@
 namespace TransIT.API.Extensions
 {
-    public static class ROLE
+    public static class RoleConsts
```
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Roles.cs]: - remove 'static'
- rename to 'Role'

File: TransIT.BLL/Helpers/Roles.cs [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-        public const string ANALYST = "ANALYST";
-        public const string WORKER = "WORKER";
+        public const string Register = "REGISTER";
+        public const string Engineer = "ENGINEER";
+        public const string Admin = "ADMIN";
+        public const string Analyst = "ANALYST";
+        public const string Worker = "WORKER";
 
-        public const string ROLE_SCHEMA = "http://schemas.microsoft.com/ws/2008/06/identity/claims/role";
+        public const string RoleSchema = "http://schemas.microsoft.com/ws/2008/06/identity/claims/role";
```
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Roles.cs]: - rename to 'Schema'

File: TransIT.BLL/Helpers/Configuration/AzureConfig.cs [hunk 3]
```diff
@@ -1,13 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace TransIT.BLL.Helpers.Configuration
+﻿namespace TransIT.BLL.Helpers.Configuration
 {
     static class AzureConfig
```
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Configuration/AzureConfig.cs]: remove 'static', add modifier

File: TransIT.BLL/Helpers/Configuration/AzureConfig.cs [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
     {
-        public static readonly string StorageAccountName = "transitdocuments";
-        public static readonly string StorageAccountkey = "pqu5uvteuvJqrcC2qLkxZH71MyV1bOPbzJKGOGTMGY73qlVxTTPNFDpkNLNH/pu6E9n7N7uJTdCya8u/TwtmEw==";
-        public static readonly string StorageConnectionString = "DefaultEndpointsProtocol=https;AccountName=transitdocuments;AccountKey=pqu5uvteuvJqrcC2qLkxZH71MyV1bOPbzJKGOGTMGY73qlVxTTPNFDpkNLNH/pu6E9n7N7uJTdCya8u/TwtmEw==;EndpointSuffix=core.windows.net";
+        public const string StorageAccountName = "transitdocuments";
```
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Configuration/AzureConfig.cs]: -remove the word 'Storage' from these constants

File: TransIT.BLL/Helpers/Warranties.cs [hunk 5]
```diff
@@ -1,8 +1,9 @@
...
 {
-    public static class Warranties
+    // needs to investigate if this enum is realy needed
+    enum Warranties : int
     {
-        public const int NON_WARRANTY_CASE = 0;
-        public const int WARRANTY_CASE = 1;
+        NonWarrantyCase = 0,
+        WarrantyCase = 1
     }
```
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Warranties.cs]: use boolean if possible, change the DAL entity as well

File: TransIT.BLL/Services/ImplementedServices/IssueService.cs [hunk 6]
```diff
@@ -50,7 +50,7 @@ public override async Task<Issue> CreateAsync(Issue issue)
         {
             Vehicle vehicle = await _vehicleRepository.GetByIdAsync(issue.VehicleId);
             if (IsWarrantyCase(vehicle))
-                issue.Warranty = Warranties.WARRANTY_CASE;
+                issue.Warranty = (int?)Warranties.WarrantyCase;
```
Comment (igolovchuk) [inline TransIT.BLL/Services/ImplementedServices/IssueService.cs]: try to change type to boolean

**Review by igolovchuk — CHANGES_REQUESTED** [review]
please look at the comments and try to resolve them

File: TransIT.BLL/Helpers/Roles.cs (hunk 1 above)
Comment (BohdanBuhrii) [reply TransIT.BLL/Helpers/Roles.cs]: If rename to "Role" then will be a conflict with Issue.Role, is this ok?

Comment (igolovchuk) [reply TransIT.BLL/Helpers/Roles.cs]: I don't see Issue.Role, where it's located?

Comment (BohdanBuhrii) [reply TransIT.BLL/Helpers/Roles.cs]: Sorry, I was wrong, I meant Entities.Role

Comment (igolovchuk) [reply TransIT.BLL/Helpers/Roles.cs]: Rename RoleConsts to RoleNames

**Review by igolovchuk — CHANGES_REQUESTED** [review]
Rename RoleConsts to RoleNames and revert last changes regarding using Namespaces to fix conflict names issue

File: TransIT.BLL/Helpers/Warranties.cs [hunk 7]
```diff
@@ -1,8 +1,9 @@
 ﻿namespace TransIT.BLL.Helpers
 {
-    public static class Warranties
+    // needs to investigate if this enum is realy needed
+    public class Warranties
     {
```
Comment (igolovchuk) [inline TransIT.BLL/Helpers/Warranties.cs]: - remove this class and in usages just set 'true' or 'false', eg. issue.Warranty = true;

File: TransIT.BLL/Services/ImplementedServices/IssueService.cs [hunk 8]
```diff
@@ -50,7 +51,7 @@ public override async Task<Issue> CreateAsync(Issue issue)
         {
             Vehicle vehicle = await _vehicleRepository.GetByIdAsync(issue.VehicleId);
             if (IsWarrantyCase(vehicle))
-                issue.Warranty = Warranties.WARRANTY_CASE;
+                issue.Warranty = Warranties.WarrantyCase;
```
Comment (igolovchuk) [inline TransIT.BLL/Services/ImplementedServices/IssueService.cs]: issue.Warranty = true;

**Review by igolovchuk — CHANGES_REQUESTED** [review]
- see comments

**Review by igolovchuk — DISMISSED** [review]

**Review by VitalikRibiy — APPROVED** [review]

**Review by igolovchuk — APPROVED** [review]

**Review by Vladacdc — APPROVED** [review]

**Comment by Vladacdc** [comment]
DO NOT MERGE UNTIL CLIENT IS READY FOR THIS MERGE

**Pull request merged**